testsuite-native-image/src/main/java/io/netty/testsuite/svm/HttpNativeServer.java

```diff
@@ -54,6 +54,7 @@ public static void main(String[] args) throws Exception {
              .childHandler(new HttpNativeServerInitializer());
 
             b.bind(PORT).sync().channel();
+            System.err.println("Server started, will shutdown now.");
```

Comment:
@pmlopes can you also call `channel.close().sync()` ?

Reply:
Done.